src/lib.rs

```diff
@@ -1,3 +1,8 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
```

Comment:
Not strictly necessary, but I will add a new line between the header and the beginning of the crates' comments, for readability.

Also, as discussed I would prefer having the license header added only when they can be tested, so please close this PR in favor of #9

Reply:
KK fair enough. makes sense.